src/ui/graph.jl

```diff
@@ -100,7 +100,7 @@ function show_ast(io, f)
 end
 
 function showfn(io, f::Function)
-    if isgeneric(f)
+    if isa(f, Function)
```

Comment:
This fix makes that feature go away on 0.4, but that's ok. I guess we have to figure out a way to do this on 0.5 then.